src/services/NotificationService.ts

```diff
@@ -0,0 +1,364 @@
+            return { 
+                success: false, 
+                reason: `Notification delivery failed: ${error.message}` 
+            };
+        }
+    }
+
+    /**
+     * Build notification message text
+     */
+    private buildNotificationMessage(pr: GitHubPullRequest, reviewer: ReviewerRecommendation, repoName: string): string {
+        const prUrl = `https://github.com/${repoName}/pull/${pr.number}`;
+        const expertiseText = reviewer.expertise.length > 0 ? ` (${reviewer.expertise.join(', ')})` : '';
+        
+        return `🔍 **You've been assigned to review a pull request!**
+
+**Repository:** ${repoName}
+**PR #${pr.number}:** ${pr.title}
+**Author:** @${pr.user.login}
+
+**Why you were selected:**
+${reviewer.reasoning}
+
+**Your expertise match:** ${reviewer.familiarityLevel}${expertiseText}
+${reviewer.codeownersMatch ? '🎯 **CODEOWNERS match**' : ''}
+${reviewer.recentActivity ? '📈 **Recent activity on these files**' : ''}
+
+**View PR:** ${prUrl}
+
+*Powered by Code Review Agent - helping you focus on the reviews that matter most.*`;
+    }
+
+    /**
+     * Build interactive notification blocks
+     */
+    private buildNotificationBlocks(pr: GitHubPullRequest, reviewer: ReviewerRecommendation, repoName: string): LayoutBlock[] {
+        const prUrl = `https://github.com/${repoName}/pull/${pr.number}`;
+        
+        const blocks: LayoutBlock[] = [
+            getSectionBlock(`🔍 **New PR Review Assignment**
+
+**Repository:** ${repoName}
+**PR #${pr.number}:** ${pr.title}
+**Author:** @${pr.user.login}
+**Match Level:** ${reviewer.familiarityLevel} (${reviewer.score}/100)`),
+            
+            getSectionBlock(`**Why you were selected:**
+${reviewer.reasoning}`),
+            
+            getActionsBlock('pr_actions', [
+                getButton({
+                    labelText: 'View Pull Request',
+                    actionId: 'view_pr',
+                    value: prUrl,
+                    style: 'primary',
+                    url: prUrl
+                }),
+                getButton({
+                    labelText: 'View Changed Files',
+                    actionId: 'view_files',
+                    value: `${prUrl}/files`,
+                    url: `${prUrl}/files`
+                })
+            ])
+        ];
+
+        // Add expertise and match indicators
+        const indicators: string[] = [];
+        if (reviewer.codeownersMatch) indicators.push('🎯 CODEOWNERS match');
+        if (reviewer.recentActivity) indicators.push('📈 Recent file activity');
+        if (reviewer.expertise.length > 0) indicators.push(`💡 Expertise: ${reviewer.expertise.join(', ')}`);
+
+        if (indicators.length > 0) {
+            blocks.push(getSectionBlock(indicators.join('\n')));
+        }
+
+        return blocks;
+    }
+
+    /**
+     * Send notification to admins about successful PR processing
+     */
+    public async notifyAdminsOfProcessing(
```

Comment:
I don't think this is required, admins can themselves rely on a spam command to see a table of contents in a modal or sidebar for the PRs processed, etc.